modules/bound-sa-tokens-configuring.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * authentication/bound-service-account-tokens.adoc
+
+[id="bound-sa-tokens-configuring_{context}"]
+= Configuring bound service account tokens using volume projection
+
+You can configure Pods to request bound service account tokens by using volume projection.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have created a service account. This procedure assumes that the service account is named `build-robot`.
+
+.Procedure
+
+. Optionally, set the service account issuer.
++
+This step is typically not required if the bound tokens are used only within the cluster.
+
+.. Edit the `cluster` authentication object.
++
+----
+$ oc edit authentications cluster
+----
+
+.. Set the `spec.serviceAccountIssuer` field to the desired service account issuer value. The default is [x-]`https://kubernetes.default.svc`.
++
+[source,yaml]
+----
+spec:
+  serviceAccountIssuer: https://test.default.svc <1>
+----
+<1> The issuer value must start with `https://`.
+
+. Configure a Pod to use a bound service account token via volume projection.
+
+.. Create a file called `pod-projected-svc-token.yaml` with the following contents:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nginx
+spec:
+  containers:
+  - image: nginx
+    name: nginx
+    volumeMounts:
+    - mountPath: /var/run/secrets/tokens
+      name: vault-token
+  serviceAccountName: build-robot <1>
+  volumes:
+  - name: vault-token
+    projected:
+      sources:
+      - serviceAccountToken:
+          path: vault-token <2>
+          expirationSeconds: 7200 <3>
+          audience: vault <4>
+----
+<1> A reference to the existing service account.
+<2> The path relative to the mount point of the file to project the token into.
+<3> Optionally set the expiration of the service account token, in seconds. The default is 3600 seconds (1 hour) and must be at least 600 seconds (10 minutes). The kubelet will start trying to rotate the token if the token is older than 80 percent of its time to live or if the token is older than 24 hours.
+<4> Optionally set the intended audience of the token. A recipient of a token must identify itself with an identifier specified in the audience of the token, and otherwise should reject the token. The audience defaults to the identifier of the API server.
```

Comment:
Change 

```
A recipient of a token must identify itself with an identifier specified in the audience of the token, and otherwise should reject the token.
```

to

```
The recipient of a token should verify that the recipient identity matches the the audience claim of the token, and should otherwise reject the token.
```